frontend2/src/components/sidebar/__test__/Sidebar.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Sidebar from "../";
+import { DEFAULT_EPISODE } from "../../../utils/constants";
+import { EpisodeContext } from "../../../contexts/EpisodeContext";
+import { MemoryRouter } from "react-router-dom";
+
+test('should link to default episode', () => {
```

Comment:
My tests are currently not working but when they are we will have dozens... thinking this may be a good opportunity to use the `API/UI: <test message> (STABLE/UNSTABLE)` format for test messages?

Reply:
```suggestion
test('UI: should link to default episode', () => {
```

Reply:
update after discussion: we won't include (stable/unstable), but we will include api/ui